### ISSUE TITLE
Simple graphical 'bug' in keybase cmd

### ISSUE THREAD

**Issue by GLugia** [keybase/keybase-issues#2312]
[When I do `keybase device list`](https://biyori.moe/d/pngnsrtjvrypy.png)

**Comment by zanderz** [comment]
Yes, this command seems to require more than 122 columns to appear correctly.

**Comment by zQueal** [comment]
Yes, that seems to be it. So less of a bug, more of a hindrance of `cmd.exe`.

![owexoc](https://cloud.githubusercontent.com/assets/239193/16430930/1b0d9814-3d4c-11e6-8bc8-1b49828e8373.png)
